app/models/manager_refresh/save_collection/base.rb

```diff
@@ -2,16 +2,16 @@ module ManagerRefresh::SaveCollection
   class Base
     class << self
       def save_inventory_object_inventory(ems, inventory_collection)
-        _log.info("Synchronizing #{ems.name} collection #{inventory_collection} of size #{inventory_collection.size} to"\
-                " the database")
+        _log.info("SYNCHRONIZING collection #{inventory_collection} of size #{inventory_collection.size} to"\
+                  " the database, for the manager: '#{ems.name}'")
 
         if inventory_collection.custom_save_block.present?
-          _log.info("Synchronizing #{ems.name} collection #{inventory_collection} using a custom save block")
+          _log.info("Synchronizing collection #{inventory_collection} using a custom save block")
```

Comment:
@Ladas can you change this to match the rest? `SYNCHRONIZING collection`

Reply:
I finally did it. 🎉